x/ecocredit/client/basket/tx_create.go

```diff
@@ -0,0 +1,207 @@
+package basketclient
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/regen-network/regen-ledger/x/ecocredit/basket"
+)
+
+const (
+	FlagDisplayName       = "display-name"
+	FlagExponent          = "exponent"
+	FlagDisableAutoRetire = "disable-auto-retire"
+	FlagCreditTypeName    = "credit-type-name"
+	FlagAllowedClasses    = "allowed-classes"
+	FlagMinimumStartDate  = "minimum-start-date"
+	FlagStartDateWindow   = "start-date-window"
+	FlagBasketFee         = "basket-fee"
+)
+
+func TxCreateBasket() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-basket [name]",
+		Short: "Creates a bank denom that wraps credits",
+		Long: strings.TrimSpace(`Creates a bank denom that wraps credits
+
+Parameters:
+		name: the name used to create a bank denom for this basket token.
+
+Flags:
+		display_name: the name used to create a bank denom display name.
```

Comment:
Can't wait till lens is fully ready and we don't need to write CLI commands anymore and copy all the docs in the .proto files manually lol